pets-project/restservice/src/main/java/com/example/restservice/pet/model/Pet.java

```diff
@@ -3,7 +3,13 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
 
+@Setter
+@Getter
+@AllArgsConstructor
```

Comment:
Why did you remove all the getters to do this? isn't it inconsistent with the rest of the tables?

Reply:
Replaced the Getter and Setter annotations for methods.